### PR TITLE
Changes to indestructible. No more immovable indestructibles

### PR DESCRIPTION
**Pull request by Nanotrasen-Spy**
## About The Pull Request

Makes it so indestructible objects can now be thrown around by explosions even without being deleted. Also a few changes so heads are indestructible so they don't get deleted when gibbed or by explosions (hooray RP hopeless synths and hooray being able to ID who inted into the center of the OB).

## Why It's Good For The Game

Makes indestructible items more "believable" instead of them being sitting fat rocks surviving multiple OB hits without moving an inch & makes it so synths have a little more to it if they get gibbed or exploded since most of the time when they lose their heads theyre for X reasons theyre deleted.

## Changelog

:cl: Nanotrasen-Spy
code: small changes to gibbing and limb objects, heads will no longer be deleted by gibbing (hooray synths)
code: Refactored explosion deletion code. Indestructible objects may also now be tossed around by explosions without being deleted.
/:cl:

### CODE REVIEW

**Comment by Watermelon914** [comment]
improve your title name, it gives barely any information right now

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -149,18 +149,17 @@
 /obj/item/ex_act(severity, explosion_direction)
 	switch(severity)
 		if(0 to EXPLOSION_THRESHOLD_LOW)
+			explosion_throw(severity, explosion_direction)
```
Comment (Geevies) [inline code/game/objects/items.dm:152]: since this is under every item of the switch, you can just put this above the switch so it always fires

**Review by Geevies — CHANGES_REQUESTED** [review]

**Comment by Watermelon914** [comment]
title is still not detailed enough. what is 'Changes to indestructible objects' supposed to mean? What have you done with indestructible objects? please include details

**Comment by fira** [comment]
Walter's right, detail both changes separate on two lines in changelog or something

Been wondering about gib decap forever, but don't think it'll cause issues (maybe synths..)

**Comment by morrowwolf** [comment]
Should have staled out. Re-open when merge conflicts are fixed and you have a coder to review it.

**Pull request closed without merging**